src/TiciTacaToeyGameEngine.ts

```diff
@@ -426,9 +423,9 @@ class TiciTacaToeyGameEngine implements GameEngine {
       case MessageTypes.MAKE_MOVE: {
         const game = this.games[message.gameId];
 
-        game.timers[message.playerId].stop();
-        game.timers[message.playerId].timeLeft +=
-          game.timers[message.playerId].increment;
+        game.timers[message.playerId].stop(
+          game.timers[message.playerId].increment
```

Comment:
This wont work as we have removed the increment property from Timer class. Can we use the following?

```typescript
game.incrementPerPlayer
```